tools/doc/tests/test_json.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+var assert = require('assert');
+var fs = require('fs');
+
+var json = require('../json.js');
+
+// Outputs valid json with the expected fields when given simple markdown
+function test_doJSON() {
+  // Test data is a list of objects with two properties.
+  // The file property is the file path.
+  // The json property is some json which will be generated by the doctool.
+  var testData = [
+    {
+      'file': 'tools/doc/tests/fixtures/sample_document.markdown',
+      'json': {
+        'source': 'foo',
+        'modules': [ { 'textRaw': 'Sample Markdown',
+                       'name': 'sample_markdown',
+                       'modules': [ { 'textRaw':'Seussian Rhymes',
+                                      'name': 'seussian_rhymes',
+                                      'desc': '<ol>\n<li>fish</li>\n<li>' +
+                                              '<p>fish</p>\n</li>\n<li>' +
+                                              '<p>Red fish</p>\n</li>\n' +
+                                              '<li>Blue fish</li>\n</ol>\n',
+                                      'type': 'module',
+                                      'displayName': 'Seussian Rhymes'
+                                   } ],
+                       'type': 'module',
+                       'displayName': 'Sample Markdown'
+                 } ]
+      }
```

Comment:
Can you 2-space indent this block, and possibly format it like the output below?